src/storage/src/storage/read_object.rs

```diff
@@ -697,6 +722,26 @@ pub struct ObjectHighlights {
     /// the client should compute one of these checksums over the downloaded
     /// object and compare it against the value provided here.
     pub checksums: std::option::Option<crate::model::ObjectChecksums>,
+
+    /// Storage class of the object.
+    pub storage_class: String,
+
+    /// Content-Language of the object data, matching
+    /// [<https://tools.ietf.org/html/rfc7231#section-3.1.3.2>][RFC 7231 §3.1.3.2].
+    pub content_language: std::string::String,
```

Comment:
nit: why `std::string::String` here vs. `String` in the previous one? Maybe cut&paste from somewhere?

Reply:
Ah thanks good catch. Was copying these from the generated crate::model::Object.